g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/dao/impl/SourceDaoImpl.java

```diff
@@ -39,172 +38,158 @@
 import com.vmware.vip.common.i18n.dto.SingleComponentDTO;
 import com.vmware.vip.common.i18n.resourcefile.LocalJSONReader;
 import com.vmware.vip.common.i18n.resourcefile.ResourceFilePathGetter;
-import com.vmware.vip.common.i18n.resourcefile.ResourceFileWritter;
 import com.vmware.vip.common.l10n.source.dto.ComponentSourceDTO;
 
 @Repository
 public class SourceDaoImpl implements SourceDao {
-	private static Logger LOGGER = LoggerFactory.getLogger(SourceDaoImpl.class);
-	@Autowired
-	private SqlLiteDao sqlLite;
-	
-	@Override
-	public String getFromBundle(SingleComponentDTO singleComponentDTO,
-			String basepath) {
-		String result = "";
-		String component = singleComponentDTO.getComponent();
-		if (StringUtils.isEmpty(component)) {
-			component = ConstantsFile.DEFAULT_COMPONENT;
-			singleComponentDTO.setComponent(component);
-		}
-		String filepath = ConstantsFile.L10N_BUNDLES_PATH
-				+ ResourceFilePathGetter
-						.getProductVersionConcatName(singleComponentDTO)
-				+ ConstantsChar.BACKSLASH
-				+ component
-				+ ConstantsChar.BACKSLASH
-				+ ResourceFilePathGetter
-						.getLocalizedJSONFileName(ConstantsKeys.LATEST);
-		LOGGER.info("Read content from file: {}{}",  basepath, filepath);
+   private static Logger LOGGER = LoggerFactory.getLogger(SourceDaoImpl.class);
+   @Autowired
+   private SqlLiteDao sqlLite;
+
+   @Override
+   public String getFromBundle(SingleComponentDTO singleComponentDTO, String basepath) {
```

Comment:
I don't understand this change. Will ping you offline.

Reply:
this just formats it and  just change the write JSON file